### PR TITLE
Duplicating iw and he values folder

### LINKED ISSUE #9451: hebrew values-he folder from crowdin needs duplication into values-iw - both are apparently needed
for working hebrew translation- please change the folder name of the hebrew translation in res to "values-iw". now it's dosn't works.
thanks

### PR DESCRIPTION
**Pull request by Aditya13s**
## Purpose / Description
The purpose of this Pull Request is to ensure that the values-iw folder is duplicated with the values-he folder.
## Fixes
* Fixes #9451

## Approach
I just added a case that if the language is `iw` then it use `androidLanguage = "heb"`

## How Has This Been Tested?
I lack the means to test this.

## Checklist

- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: tools/localization/src/update.ts [hunk 1]
```diff
@@ -197,6 +197,10 @@ export async function updateI18nFiles() {
                 androidLanguage = "heb";
                 break;
 
+            case "iw":
+                androidLanguage = "heb";
+                break;
```
Comment (david-allison) [inline tools/localization/src/update.ts:202]: `iw` isn't in the list of languages

**Review by david-allison — CHANGES_REQUESTED** [review]
> I lack the means to test this.

This affects our automated translation sync. It's not okay, especially as this change wouldn't work

You need to get this to the point that it's testable outside of production, start by looking at CI, then by contacting the author if you haven't been able to run the script

**Comment by Aditya13s** [comment]
> > I lack the means to test this.
> 
> This affects our automated translation sync. It's not okay, especially as this change wouldn't work
> 
> You need to get this to the point that it's testable outside of production, start by looking at CI, then by contacting the author if you haven't been able to run the script

`iw` works on old devices and i don't have any old device to test and I have doubt that do i have to manually copy `he` files in `iw` files

**Comment by david-allison** [comment]
The automated copying process needs to be tested

**Review by david-allison — CHANGES_REQUESTED** [review]
Don't bother with the XML, this is about fixing the translation sync script

We don't want a manual step in the build process

**Comment by Aditya13s** [comment]
> Don't bother with the XML, this is about fixing the translation sync script
> 
> We don't want a manual step in the build process

I removed the xml. I have no idea about what to change in the script. Please help me with that.

**Comment by david-allison** [comment]
I don't have the capacity for that currently, sorry

**Comment by BrayanDSO** [comment]
This PR is far from what the task needs, and the author themselves said that they don't know how to fix it. 

So, there's nothing to review here.

**Pull request closed without merging**